user-service/controller/user-controller.js

```diff
@@ -199,12 +177,13 @@ const generateToken = (username) => {
         const userToken = {
             username: username
         }
-        return jwt.sign(userToken, process.env.SECRET)
+        return jwt.sign(userToken, process.env.SECRET, {expiresIn: '2h'})
     } catch (err) {
         console.log('ERROR: Could not generate token');
     }

```

Comment:
Throw to frontend as well on top of console.log?

Reply:
Main function loginUser will throw error to frontend 
![image](https://user-images.githubusercontent.com/77485021/193451052-50e1bbad-c7a5-44c3-b264-464f3c48b599.png)